### ISSUE TITLE
Using current as part of string value

### ISSUE THREAD

**Issue by sebastiaanviaene** [RobinCK/typeorm-fixtures#26]
I was wondering if it's possible to use the current iteration as a string value in a field instead of just referencing relations. Like this:

````
entity: User
items:
  user{1..10}:
    firstName: '{{name.firstName}}'
    lastName: '{{name.lastName}}'
    email: 'email{{current}}@gmail.com'
````

This way all the fixtures will have email1@gmail.com -> email10@gmail.com

**Comment by sebastiaanviaene** [comment]
#30

**Comment by RobinCK** [comment]
Thank @sebastiaanviaene 
implemented in typeorm-fixtures-cli@1.2.0
